src/util/docker.rs

```diff
@@ -72,7 +72,7 @@ impl DockerCommand {
             env_file,
             name: None,
             daemon: false,
-            tty: false,
+            tty: true,
```

Comment:
This change will effect the behavior of:
* capsule debugger start (enable `tty` only when start gdb server)
* run CI job in github action, since can't get a tty.

Suggest give an option in command arguments.

Reply:
I prefer auto-detecting if the user is running at the tty mode than an option arg, I added a commit which is using `atty` crate to resolve issue, this crate has been used by `clap` already, please review again, thanks.